addOns/requester/src/main/java/org/zaproxy/zap/extension/requester/ExtensionRequester.java

```diff
@@ -81,7 +90,23 @@ public static ImageIcon getRequesterIcon() {
     public void hook(ExtensionHook extensionHook) {
         super.hook(extensionHook);
         extensionHook.addOptionsParamSet(getOptionsParam());
+
+        addHrefType(
+                extensionHook,
+                new HrefTypeInfo(
+                        HistoryReference.TYPE_ZAP_USER,
+                        Constant.messages.getString("view.href.type.name.manual"),
+                        hasView() ? getManualIcon() : null));
+
         if (getView() != null) {
```

Comment:
Could use `hasView()`?

Reply:
I don't usually update other code unless the patch touches it. IMO we should do these updates in their own PRs and change everything at once.